### DISCUSSION TITLE
Scipy citation

### DISCUSSION THREAD

**Discussion by prajwel** [astropy/astropy-v5.0-paper#107] [Text comments] [answered]
The citation to Scipy is presently the following. 

Jones, E., Oliphant, T., Peterson, P.,
et al. 2001–, SciPy: Open source
scientiﬁc tools for Python.
http://www.scipy.org/

But "citing Scipy" web page (https://scipy.org/citing-scipy/) recommends the following. 

Pauli Virtanen, Ralf Gommers, Travis E. Oliphant, Matt Haberland, Tyler Reddy, David Cournapeau, Evgeni Burovski, Pearu Peterson, Warren Weckesser, Jonathan Bright, Stéfan J. van der Walt, Matthew Brett, Joshua Wilson, K. Jarrod Millman, Nikolay Mayorov, Andrew R. J. Nelson, Eric Jones, Robert Kern, Eric Larson, CJ Carey, İlhan Polat, Yu Feng, Eric W. Moore, Jake VanderPlas, Denis Laxalde, Josef Perktold, Robert Cimrman, Ian Henriksen, E.A. Quintero, Charles R Harris, Anne M. Archibald, Antônio H. Ribeiro, Fabian Pedregosa, Paul van Mulbregt, and SciPy 1.0 Contributors. (2020) SciPy 1.0: Fundamental Algorithms for Scientific Computing in Python. Nature Methods, 17(3), 261-272.

**Comment by pllim** [comment, marked as answer]
Does #117 look good to you? Thanks for bringing this to our attention!
